include/deal.II/base/utilities.h

```diff
@@ -17,40 +17,40 @@
 #define dealii_utilities_h
 
 #include <deal.II/base/config.h>
+
 #include <deal.II/base/exceptions.h>
 
-#include <vector>
-#include <utility>
 #include <functional>
 #include <string>
 #include <tuple>
 #include <type_traits>
+#include <utility>
+#include <vector>
 
 #ifdef DEAL_II_WITH_TRILINOS
-#  include <Epetra_Comm.h>
-#  include <Epetra_Map.h>
-#  ifdef DEAL_II_WITH_MPI
-#    include <Epetra_MpiComm.h>
-#  else
-#    include <Epetra_SerialComm.h>
-#  endif
+#include <Epetra_Comm.h>
+#include <Epetra_Map.h>
```

Comment:
Oh, that's a pity -- these indentations make this a lot easier to read...

Reply:
That is one thing I don't know how to teach clang-format how to do which I already miss.

Reply:
This is `IndentPPDirectives`.